integration/eslint_test.js

```diff
@@ -1,7 +1,9 @@
 const sinon = require("sinon")
-const expect = require("chai").expect
+const chai = require("chai")
+const expect = chai.expect
+chai.use(require('chai-as-promised'))
```

Comment:
wdyt about adding a note on the  pr description about this new project dependency and an example of why it's handy for testing ?